buildconfig.js

```diff
@@ -41,12 +49,13 @@ if (reConfig.test(indexHtml)) {
 
 // Replace ZenDesk Javascript
 if (typeof process.env.HELP_LINK === 'string') {
+  console.log("Using HELP_LINK: ", process.env.HELP_LINK);
+
   if (process.env.HELP_LINK === 'disable') {
-    indexHtml = indexHtml.replace(/(<script id="ze-snippet".*<\/script>)/, '');
+    indexHtml = indexHtml.replace(reZendesk, zendeskDisable);
+
   } else {
-    indexHtml = indexHtml.replace(/(<script id="ze-snippet" src="([^"]+)">)/, (match, p1, p2) => {
-      return p1.replace(p2, process.env.HELP_LINK);
-    });
+    indexHtml = indexHtml.replace(reZendesk, `<script type="text/javascript" src="${process.env.HELP_LINK}"></script>`);
```

Comment:
When script is present, the pattern for search (<script id="ze-snippet"...) doesn't match the replacement (<script type="text/javascript"...). 
We won't be able to apply an HELP_LINK change twice.
```suggestion
    indexHtml = indexHtml.replace(reZendesk, `<script id="ze-snippet" src="${process.env.HELP_LINK}"></script>`);
```